scripts/set_version_to.py

```diff
@@ -54,7 +54,7 @@
                     data[1] = args.text
                     line = '"'.join(data)
             o += line
-        outfile = open(filename, "w")
+        outfile = open(filename, "w", newline='\n')
```

Comment:
👍 not attempting to cure the symptom any longer

Reply:
Tough it is not the source of
```
warning: in the working copy of 'CommonAssemblyInfoExternals.cs', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'Externals/NetSpell.SpellChecker/Properties/AssemblyInfo.cs', LF will be replaced by CRLF the next time Git touches it
```

Reply:
As text=auto in .gitattributes, .cs are converted crlf<->lf
Maybe setting .xlf as text instead?